firefox-webext/background.js

```diff
@@ -2,9 +2,98 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+var fetchURL = 'https://api.github.com/repos/webcompat/web-bugs/issues?state=open&page=1&per_page=100';
+var issuesList = [];
+var requestIssues = [];
+var ignoreList = ['about'];
```

Comment:
I guess we want to include other things like `file://`, `view-source://`, `resource://`, etc. Maybe everything that isn't http or https?